test/iteration.jl

```diff
@@ -19,18 +15,34 @@ module TestIteration
     end
 
     @test size(eachcol(df)) == (size(df, 2),)
+    @test size(columns(df)) == (size(df, 2),)
     @test length(eachcol(df)) == size(df, 2)
-    @test eachcol(df)[1] == df[1]
-    @test collect(eachcol(df)) isa Vector{Tuple{Symbol, AbstractVector}}
-    @test eltype(eachcol(df)) == Tuple{Symbol, AbstractVector}
+    @test length(columns(df)) == size(df, 2)
+    @test eachcol(df)[1] == df[1] # this will be (:A => df[1]) after deprecation
+    @test columns(df)[1] == df[1]
+    @test collect(eachcol(df)) isa Vector{AbstractVector} # this will be Vector{Pair{Symbol, AbstractVector}} after deprecation
+    @test collect(eachcol(df)) == [:A => Union{Missing, Int64}[1, 2]
```

Comment:
`Union{Missing, Int64}` isn't needed.

Reply:
OK. I will leave that out, but I thought it does not hurt either 😄.